### DISCUSSION TITLE
Mail sender node

### DISCUSSION THREAD

**Discussion by Ibrahim-Halil-Kuray** [ProxeusApp/community#24] [Q&A] [answered]
Mail node sender is not working. It is unclear from which variables the information is obtained and how the information is obtained.
For example, it is specified how the variable should be named for the Price retriever node. How the information is transmitted to the node is understandable, but no such explanation and usage is specified for the Email Sender node.

See also: [screenshot of error](https://user-images.githubusercontent.com/71791007/235914930-b0655907-9a48-42e5-bb56-780a647523f7.png) "node impl missing".

**Comment by loleg** [comment, marked as answer]
This node is part of [proxeus-core](https://github.com/ProxeusApp/proxeus-core/), and the documentation really is lacking some details or an example usage. Let's make this an issue to update the documentation.
